base.scss

```diff
@@ -0,0 +1,68 @@
+h1 {
+    font-family: Lucida Sans Unicode;
+    border-bottom: 1px solid silver;
+}
+
+#page-wrapper {
+    width: 500px;
+    position: absolute;
+    top: 40px;
+    left: 50%;
+    margin-left: -250px;
+    padding: 5px;
+}
+
+#heading {
+    font-size: 24px;
+    text-align: center
+}
+
+#errors {
+    border: 1px solid #777777;
+    padding: 3px;
+    display: none;
+}
+
+.row {
+    margin: 15px 0;
+}
+
+.row label {
+    display: inline-block;
+    float: left;
+    width: 150px;
+    font-family: Lucida Sans Unicode;
+}
+
+.row input[type=text] {
+    width: 200px;
+    padding: 3px;
+    border: 1px solid #777777;
+}
+
+textarea {
+    width: 200px;
+    padding: 3px;
+    border: 1px solid #777777;
+}
+
+.clearfloat {
+    clear: both;
+    height: 0;
+    font-size: 1px;
+    line-height: 0px;
+}
+
+.large {
+    font-size: 24px;
+}
+
+#submit {
+    color: #333;
+    text-decoration: none;
+    display: inline-block;
+    font-family: Lucida Sans Unicode;
+    padding: 4px 8px;
+    background: #7EB87E;
+    border: 1px solid #4D814D;
+}
```

Comment:
Initial CSS file formatting was not readable - all css rules in one line for each selector. Each css rule should be in it's own line.